benchcab/data/test/integration.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Should we set the file to be executable? Prior to running the script, I run `chmod +x benchcab/data/test/integration.sh`.

Reply:
This isn't a huge issue, so I'm happy either way (i.e. regular file vs executable).